Source/Rhetos.Utilities/ApplicationConfiguration/ConfigurationSources/JsonFileSource.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Rhetos.Utilities.ApplicationConfiguration.ConfigurationSources
+{
+    public class JsonFileSource : IConfigurationSource
+    {
+        private readonly string _filePath;
+
+        public JsonFileSource(string filePath)
+        {
+            _filePath = Path.GetFullPath(filePath);
+        }
+
+        public IDictionary<string, object> Load()
+        {
+            if (!File.Exists(_filePath))
+                throw new FrameworkException($"Specified Json configuration file '{_filePath}' doesn't exist.");
+
+            string json;
+            try
+            {
+                json = File.ReadAllText(_filePath);
+            }
+            catch (Exception e)

```

Comment:
The first two exception might be removed. ReadAllText method returns the same information (Could not find file 'C:\temp\someconfiguration.json'.).
The last try..cache is still needed because JsonSource does not have the _filePath information.